frontend/src/components/ProfileContent.tsx

```diff
@@ -165,8 +175,32 @@ export default function ProfileContent({
           <Grid item xs={12} sm={6}>
             <Avatar src={""} className={classes.profileAvatar} />
           </Grid>
-          <Grid item xs={12} sm={6}>
-            <div className={classes.introVideo}>Intro Video</div>
+          <Grid
+            item
+            xs={12}
+            sm={6}
+            container
+            alignItems="center"
+            justify="center"
+            className={classes.introVideo}
+          >
+            {readonlyContent.video ? (
+              <video
+                controls
+                src={readonlyContent.video}
+                width={"100%"}
+                height={"100%"}
+              />
+            ) : (
+              <Button
+                className={classes.recordButton}
+                size="large"
+                component={Link}
+                to="/record"
```

Comment:
will the record page redirect back to profile?

Reply:
I could add that in